web-client/app/scripts/filters/timeToRead.js

```diff
@@ -3,15 +3,14 @@
 angular.module('webClientApp')
   .filter('timeToRead', [function () {
     return function(text) {
```

Comment:
I would change the variable to time to indicate what is expected. This filter no longer expects text, rather it is expecting a number.

function(time) {
  time = +time;
  if (time === 1) {...
  }...